onnx_tf/backend.py

```diff
@@ -337,34 +316,52 @@ def onnx_initializer_to_input_dict_items(cls,
     def tensor2list(onnx_tensor):
       # Use the onnx.numpy_helper because the data may be raw
       return onnx.numpy_helper.to_array(onnx_tensor).flatten().tolist()
+
     input_dict = [(tp.name, tf.constant(tensor2list(tp),
                                         shape=tp.dims,
                                         dtype=ONNX_TYPE_TO_TF_TYPE[
-                                            tp.data_type]))
+                                          tp.data_type]))
                   for tp in initializer]
     return input_dict
 
   @classmethod
-  def op_name_to_lower(cls, name):
-    return re.sub('(?<!^)(?=[A-Z])', '_', name).lower()
+  def _onnx_node_to_tensorflow_op(cls, node, input_dict, opset=0):
+    """
+    Convert onnx node to tensorflow op.
 
-  @classmethod
-  def _onnx_node_to_tensorflow_op(cls, node, input_dict):
-    op_name_lowered = cls.op_name_to_lower(node.op_type)
+    Args:
+      node: Onnx node object.
+      input_dict: Inputs dict of graph.
+      opset: Opset version of the operator set. Default 0 means using latest version.
+
+    Returns:
+      Tensorflow op
+    """
+    op_name_lowered = op_name_to_lower(node.op_type)
     handler_name = "handle_" + op_name_lowered
 
     # Check if specialized handler exists.
-    if handler_name in dir(cls):
-      method_to_call = getattr(cls, handler_name)
+    versions = backend_opset_version[op_name_lowered]
+
+    if opset == 0:
+      version = max(versions)
+    else:
+      versions = sorted(versions + [opset])
+      version = versions[max([i for i, v in enumerate(versions) if v == opset]) - 1]
+
+    backend = importlib.import_module('onnx_tf.backends.backend_v{}'.format(version)).TensorflowBackend
```

Comment:
I think we should load and cache all versions of the backend as a class variable. Otherwise having a disk access every time we convert an op is not very efficient.